docs/tutorials/SpectralFit.ipynb

```diff
@@ -13683,99 +3382,251 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig,ax = plt.subplots()\n",
     "\n",
-    "for i, e in enumerate(energy):\n",
-    "    flux_inj[i] = K_inj * band_megalib(e, alpha_inj, beta_inj, xp_inj)\n",
+    "ax.plot(energy, energy*energy*flux_median, label = \"Best fit\")\n",
+    "ax.fill_between(energy, energy*energy*flux_lo, energy*energy*flux_hi, alpha = .5, label = \"Best fit (errors)\")\n",
```

Comment:
Comparing the fitted results with the data counts is a good idea. However, instead of scaling  the injected counts by the an approximate exposure, you can instead compare the counts directly with the expected counts as obtained from the best-fit spectrum. This would have to go into a second plot, since the y axis will be the integrated counts (not flux). 

The advantages of doing it this way are:
- You can quantitatively obtain a goodness of fit, for example, using a chi2 test.
- The expected counts would be exact, since you'll be using all the machinery in cosipy that convolves the spectrum with the detector response. Your approximation is sound, and very good for this example, but it doesn't work in general when the orientation is changing --like for the Crab.